libs/ui/src/modal.tsx

```diff
@@ -86,6 +97,7 @@ interface Props {
   actions?: ReactNode;
   onAfterOpen?: () => void;
   onOverlayClick?: () => void;
+  modalWidth?: ModalWidth;
```

Comment:
could also just take a `style` prop and apply it to the content div - might be more flexible for future style tweaks